config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,6 @@
+Sidekiq.configure_server do |config|
+    config.redis = { url: 'redis://redis:6379/0', password: 'yourpassword' }
```

Comment:
Maybe the url and password should go in the secrets file?

Reply:
maybe, I did it the way it is in the documentation